esmvaltool/utils/cmorizers/obs/cmor_config/OSI-409-nh.yml

```
@@ -0,0 +1,23 @@
+
+---
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: OSI-409-nh
+  version: 'v1.2'
+  tier: 2
+  modeling_realm: reanaly
+  project_id: CMIP5
```

```suggestion
  project_id: OBS
```
should be the same as project in the recipe

---

Done